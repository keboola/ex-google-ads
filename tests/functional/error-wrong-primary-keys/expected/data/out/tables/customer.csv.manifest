{
    "primary_key": [
        "id"
    ],
    "columns": [
        "id",
        "descriptiveName",
        "currencyCode",
        "timeZone"
    ],
    "incremental": true
}
