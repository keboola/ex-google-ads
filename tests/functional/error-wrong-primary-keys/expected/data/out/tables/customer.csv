"9250499593","Keboola Test","CZK","Europe/Prague"